notebooks/GP-Marginal.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
<ul><li><s>MareginalGP</s> <strong>Marginal GP</strong> Model</li><li>It contains <strong>the</strong> <code>marginal_likelihood</code> and <code>conditional</code> method<strong>s</strong>, that do<s>es</s> exactly <s>as</s> <strong>what's</strong> described in the previous section. Moreover, it also has <strong>the</strong> methods <code>predict</code> and <code>predictt</code> to...</li><li>sample from the conditional distribution and to get point estimate ... of the conditional distribution respectively: I think a word is missing at the end, otherwise the "respectively" isn't needed, as we're only talking about the conditional distribution.</li><li>Let's see each method in detail in <strong>the</strong> following sections</li><li>#Now, we can <s>instantiata</s> <strong>instantiate</strong> the model</li><li>"As <code>noise</code> behaves exactly as <code>jitter</code>, it is recommended to set <code>jitter=False</code> to avoid adding extra noise": Is it <code>jitter=False</code> or <code>jitter=0</code> (you talk about both above)?</li><li>I would explain more about how passing a covariance object as noise to the <code>marginal_likelihood</code> method is different than just passing a scalar. I'd also explain more what kind of statistical object <code>marginal_likelihood</code> is returning conceptually. This would also shed light about cases where <code>y</code> is not the observed data.</li><li>... and <code>y_pred</code> is a <strong>realization of a</strong> multivariate normal distribution.</li><li><strong>The</strong> <code>given</code> dictionary is optional when <strong>the</strong> <code>marginal_likelihood</code> method has been called before.</li><li>"To add noise in the conditional distribution, use": here also, it would be helpful to explain why adding noise is useful</li><li>"To avoid reparametrizing to the <code>MvNormalCholesky</code> distribution, use": it would be helpful to explain why the Cholesky decomposition is useful and used by default under the hood</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc4/pull/309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>